src/main/resources/assets/iris/lang/en_us.json

```diff
@@ -2,5 +2,14 @@
   "iris.shaders.reloaded": "Shaders Reloaded!",
   "iris.keybind.reload": "Reload Shaders",
   "iris.keybinds": "Iris",
-  "iris.shaders.reloaded.failure": "Failed to Reload Shaders! Reason: %s"
-}
\ No newline at end of file
+  "iris.shaders.reloaded.failure": "Failed to Reload Shaders! Reason: %s",
+
+  "options.iris.apply": "Apply",
+  "options.iris.refresh": "Refresh",
+  "options.iris.openShaderPackFolder": "Open Shader Pack Folder...",
+  "options.iris.refreshShaderPacks": "Refresh Shader Packs",
+  "options.iris.shaderPackSelection": "Shader Packs...",
+  "options.iris.shaderPackSelection.title": "Shader Packs",
+
+  "pack.iris.select.title": "Select"
```

Comment:
I think it's a bit inconsistent to have some translation strings named "iris.something.something" and others "something.iris.something", so it'd be better to rename "options.iris.apply" to "iris.options.apply" for example.

Reply:
That would be inconsistent with much of Minecraft's naming scheme, for example: `item.minecraft.gold_ingot`
Minecraft options are simply keyed with `options.<option>` so I thought it would be cleaner and more consistent to keep the `option.` prefix. I'll need another opinion on this.

Reply:
Sorry for picking the wrong line, this was my first pr review

Reply:
> That would be inconsistent with much of Minecraft's naming scheme, for example: `item.minecraft.gold_ingot`
> Minecraft options are simply keyed with `options.<option>` 

Ah, makes sense. Minecraft's translation format is weird.

Reply:
It's highly likely `item.minecraft` exists to highlight the namespace as well, and not used globally. In my opinion mod ID should always go first.

Reply:
To extend this, I believe translations built off vanilla options should use vanilla's format, while others use the `modid.category.option` format